assignments/callbacks.js

```diff
@@ -27,24 +27,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, function(length) {
+  console.log(length)
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  const end = arr.length - 1;
+  return cb(arr[end]);
 }
+last(items, function(giveLast) {
+  console.log(giveLast)
+});
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
+sumNums(5, 4, function(math) {
+  console.log(math)
+});
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
+multiplyNums(5, 4, function(mult) {
+  console.log(mult)
+});
+
+
+const stuff = ['pen', 'notes', 'string', 'mint'];
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i = 0; i < list.length; i++) {
+    if (list[i] === item) {
+      return cb('true');
+    } else {
+      return cb('false');
+    }
+  }
```

Comment:
real close here. the problem is that you have your else inside your for loop. that means that if the first item in the array is not the item you're searching for it terminates the for loop and returns false. but what if your item is actually the second item? return true in your if and just omit the else, which will allow the for loop to run again for the next item. then put the return false outside your for loop. it will only reach outside the for loop if nothing inside the for loop ever returned true, therefore itll be false